app/controllers/saml_idp_controller.rb

```diff
@@ -109,6 +110,23 @@ def capture_analytics
     analytics.track_event(Analytics::SAML_AUTH, analytics_payload)
   end
 
+  def log_external_saml_auth_request
+    return unless external_saml_request?
+
+    analytics_payload = {
+      idv: identity_needs_verification?,
+      finish_profile: profile_needs_verification?,
+      requested_ial: saml_request&.requested_ial_authn_context || 'none',
+      service_provider: saml_request&.issuer,
+    }
+    analytics.saml_auth_request(**analytics_payload)
```

Comment:
I think we can inline this, no need for `**`
```suggestion
    analytics.saml_auth_request(
      idv: identity_needs_verification?,
      finish_profile: profile_needs_verification?,
      requested_ial: saml_request&.requested_ial_authn_context || 'none',
      service_provider: saml_request&.issuer,
    )
```

Reply:
good call, applied this change in 0e2c35e37 since it won't let me commit the suggestion with the other changes